editor/src/components/inspector/inspector.tsx

```diff
@@ -373,6 +373,7 @@ export const Inspector = betterReactMemo<InspectorProps>('Inspector', (props: In
       return (
         <React.Fragment>
           <AlignmentButtons numberOfTargets={instancePaths.length} />
+          {anyComponents ? <ComponentSection isScene={false} /> : null}
```

Comment:
This probably shouldn't be in this PR... is there a separate PR open for it?

Reply:
This was just merged in another PR